RoddyCore/src/de/dkfz/roddy/core/ProjectLoader.groovy

```diff
@@ -236,10 +257,11 @@ class ProjectLoader {
             } else {
                 configurationFile << configurationText
             }
-            return
+        } finally {
+            // Unlock and close the file
+            flock.release()
```

Comment:
Ok, I'll put in a more sophisticated comment. The idea behind channel locks is, that the lock is JVM wide and therefore shared across all JVM instances on a single machine. Not more not less. It will just prevent the current user from mutual file access / file overrides. The method is platform dependent. flock is not a tool but just a variable name which means fileLock. That's it.